src/de/jost_net/JVerein/server/EigenschaftenNode2.java

```diff
@@ -0,0 +1,307 @@
+/**********************************************************************
+ * Copyright (c) by Heiner Jostkleigrewe
+ * This program is free software: you can redistribute it and/or modify it under the terms of the 
+ * GNU General Public License as published by the Free Software Foundation, either version 3 of the 
+ * License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,  but WITHOUT ANY WARRANTY; without 
+ *  even the implied warranty of  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See 
+ *  the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program.  If not, 
+ * see <http://www.gnu.org/licenses/>.
+ * 
+ * heiner@jverein.de
+ * www.jverein.de
+ **********************************************************************/
+package de.jost_net.JVerein.server;
+
+import java.rmi.RemoteException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.StringTokenizer;
+
+import de.jost_net.JVerein.Einstellungen;
+import de.jost_net.JVerein.rmi.Eigenschaft;
+import de.jost_net.JVerein.rmi.EigenschaftGruppe;
+import de.jost_net.JVerein.rmi.Eigenschaften;
+import de.jost_net.JVerein.rmi.Mitglied;
+import de.willuhn.datasource.GenericIterator;
+import de.willuhn.datasource.GenericObject;
+import de.willuhn.datasource.GenericObjectNode;
+import de.willuhn.datasource.pseudo.PseudoIterator;
+import de.willuhn.datasource.rmi.DBIterator;
+
+public class EigenschaftenNode2 implements GenericObjectNode
+{
+
+  private EigenschaftenNode2 parent = null;
+
+  private Mitglied mitglied = null;
+
+  private EigenschaftGruppe eigenschaftgruppe = null;
+
+  private Eigenschaft eigenschaft = null;
+
+  private Eigenschaften eigenschaften = null;
+
+  private ArrayList<GenericObjectNode> childrens;
+
+  private String preset = UNCHECKED;
+
+  public static final int NONE = 0;
+
+  public static final int ROOT = 1;
+
+  public static final int EIGENSCHAFTGRUPPE = 2;
+
+  public static final int EIGENSCHAFTEN = 3;
+
+  private int nodetype = NONE;
+
+  private Map<String, String> eigenschaftids = new HashMap<>();
```

Comment:
Umbenennung zu `eigenschaftIDs`?

Reply:
Die Map wird in meiner nächsten Übergabe gelöscht. Da muss man nichts mehr machen.